React/Base/RCTUtils.m

```diff
@@ -733,8 +733,13 @@ BOOL RCTIsLocalAssetURL(NSURL *__nullable imageURL)
   return [directory stringByAppendingPathComponent:filename];
 }
 
-static void RCTGetRGBAColorComponents(CGColorRef color, CGFloat rgba[4])
+void RCTGetRGBAColorComponents(CGColorRef color, CGFloat rgba[4])
```

Comment:
It's inexpensive, almost never showed up at 1ms sampling on my iPhone 6 so I'd stick with the CGColor.